components/Dialogs/examples/supplemental/DialogDismissalOverPresentedControllerExampleViewController.m

```diff
@@ -37,13 +39,16 @@ @interface PresentedViewControllerWithDialog : UIViewController
 
 @property(nonatomic, strong) UILabel *bottomRightLabel;
 
+@property(nonatomic, strong) id<MDCContainerScheming> containerScheme;
+
 @end
 
 @implementation PresentedViewControllerWithDialog
 
 - (void)viewDidLoad {
   [super viewDidLoad];
 
+  self.containerScheme = [[MDCContainerScheme alloc] init];
```

Comment:
See previous comment about consistent approach to initializing the containerScheme.

Reply:
Same as above.